vendor/vendor.json

```diff
@@ -1077,7 +1077,7 @@
 			"versionExact": "v1.0.0"
 		},
 		{
-			"checksumSHA1": "GENxfNGiSzB9hzo2fPZkI4F/Zzg=",
+			"checksumSHA1": "me40nSy9NBr9MmAwe5vDvTgSVZs=",
```

Comment:
does this patch need to update Go dependencies ? if not mandatory I would prefer to not update